### ISSUE TITLE
Subscription v1alpha1 is not transformed in v2alpha1  correctly (scopes field is missing)

### ISSUE THREAD

**Issue by fjvela** [dapr/dapr#3914]
## In what area(s)?

> /area runtime

/area operator

> /area placement

> /area docs

> /area test-and-release

## What version of Dapr?

1.5.0 / 1.4.3

## Expected Behavior
When Dapr transforms a Subscription v1alpha1  in v2alpha1 scope field must be created

## Actual Behavior
When Dapr transforms a Subscription v1alpha1  in v2alpha1 scope field is not created
![image](https://user-images.githubusercontent.com/5806680/142266676-e9f57a1c-5146-4ce8-b9a9-84e1ba2332eb.png)

## Steps to Reproduce the Problem

1. Start a new cluster with minikube
2. Install Dapr with helm
3. Create this subscription in the cluster

``` yaml
apiVersion: dapr.io/v2alpha1
kind: Subscription
metadata:
  name: node-subscriber-a
spec:
  topic: A
  routes:
    default: /A
  pubsubname: pubsub
scopes:
- nodesubscriber
---
apiVersion: dapr.io/v1alpha1
kind: Subscription
metadata:
  name: node-subscriber-b
spec:
  topic: B
  route: /A
  pubsubname: pubsub
scopes:
- nodesubscriber
```

## Release Note

RELEASE NOTE: FIX Subscription transform v1alpha1 to v2alpha1 (scopes)

**Comment by yaron2** [comment]
cc @pkedy @artursouza 

If this is correct, it'll warrant a hot fix most likely.

**Comment by pkedy** [comment]
@fjvela @yaron2 Can confirm this is correct. Thanks for reporting this! I opened a PR against the 1.5 release branch. It was only a 1 line fix but probably does warrant a hotfix because it means the CRD will be picked up by all applications.

**Comment by fjvela** [comment]
@pkedy keep in mind fix it also in 1.4.3 :)

**Comment by pkedy** [comment]
@fjvela Yes, absolutely! Thanks!
